<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>616</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>-</string>
       </property>
       <item>
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Квадрат</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Прямоугольник</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Параллелограмм</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ромб</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Треугольник</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Трапеция</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Круг</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Сектор</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="PictureLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="FormulaLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="arg1_label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>lll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="arg1_spinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="arg2_label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="arg2_spinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="arg3_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="arg3_spinBox"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="err_label">
       <property name="text">
        <string>err</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculateResult">
       <property name="text">
        <string>Вычислить результат</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="result_label">
       <property name="text">
        <string>Результат:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAuthors"/>
   <addaction name="actionClear"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionAuthors">
   <property name="text">
    <string>Авторы</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
